### DISCUSSION TITLE
Next CP2K Developers Meeting

### DISCUSSION THREAD

**Discussion by robertschade** [cp2k/cp2k#3493] [General]
Dear @cp2k/cp2k-core-developers @cp2k/cp2k-developers,
we would like to schedule our next CP2K Dev meeting (see also https://www.cp2k.org/dev:meetings) soon. We would be happy if you could fill in your availability in the survey below by the end of next week (28 th of Juli).

https://terminplaner6.dfn.de/de/p/e4192688796492ae48b098b7d8f7d3eb-781514

We would also like to invite everyone to contribute topics to the meeting. You can either comment here, let me know, or simply let us know during the meeting. If you would like to show some slides, you can either add your content to
https://docs.google.com/presentation/d/1ZblD6WNEJEV4EsfJOmpvMvC_Nxzh0-62ePolLYuR3n4/edit?usp=sharing
(please let me know if you need access)
or show your slides during the meeting with screen sharing.
Many Thanks
Robert

**Comment by gsitaram** [comment]
Hi @robertschade , if I am able to attend this meeting, I'd like to briefly talk about the benefits of unified memory programming model for APUs like MI300A and solicit ideas from the developer community for further enhancing CP2K using this approach. Please let me know if that would be okay.
cc: @mtaillefumier @alazzaro

**Reply by robertschade** [reply]
I think that would be ok. Please don't talk too long; the relation to CP2K should be clear.

**Reply by gsitaram** [reply]
Thanks, I would like a maximum of 10 mins. Please feel free to put me on the agenda where appropriate. I will be attending the whole meeting.

**Comment by rathann** [comment]
Could you remove me from the team? I no longer maintain cp2k package in Fedora. I'd like to take this opportunity to thank the team publicly for great cooperation with and hospitality towards downstream packager such as myself over the years.

**Reply by robertschade** [reply]
Hi @rathann, many thanks! (@oschuett, could you please remove him from the cp2k-developers-team?)

**Reply by oschuett** [reply]
@rathann, I've removed you from the list. Also from my side a big thank you for packaging CP2K (and many of its dependencies) for all those those years and for passing on the baton by finding a great successor!

**Comment by robertschade** [comment]
Dear @cp2k/cp2k-developers @cp2k/cp2k-core-developers ,
we would like to invite you to the next CP2K developers Meeting on July 22nd, 14:00-16:00 CEST in
https://uni-paderborn-de.zoom-x.de/j/65141261996?pwd=OHXsgAcOcXmJsd98x4b6BTY7d7agVE.1
Meeting-ID: 651 4126 1996
Kenncode: 094269

Please feel free to add topics, questions, or slides to the meeting notes at
https://docs.google.com/presentation/d/1ZblD6WNEJEV4EsfJOmpvMvC_Nxzh0-62ePolLYuR3n4/edit?usp=sharing
beforehand or comment here.
(Because this post is publically visible, I can't post here the edit-link for the slides, but simply comment or send me a mail, and I will enable edit access for you. You can also send me the slides or topics/questions that you would like to have added there.)

You can also find notes from previous meetings at https://www.cp2k.org/dev:meetings.

**Reply by gsitaram** [reply]
@robertschade, could you provide edit link to me to add my slides regarding programming model for MI300A?

**Reply by robertschade** [reply]
HI, I have added you as an editor to the slides. You should have gotten a notification mail from google.

**Reply by gsitaram** [reply]
Thank you, @robertschade! Here is a PDF of my slides if you would like to distribute. I can also add to the existing slide deck if needed. 
[Benefits_of_MI300A_Unified_Memory_Programming_Model_for_CP2K.pdf](https://github.com/user-attachments/files/16335024/Benefits_of_MI300A_Unified_Memory_Programming_Model_for_CP2K.pdf)

**Reply by hfp** [reply]
There was a question about node-scaling on slide 10, and I made a note about LLC/L3 cache effect. Due to strong-scaling or dividing work into smaller pieces, it can happen for some working set to stay in LLC. This might emphasize again how the CPU is relevant in case of CP2K even when offloading work to GPUs (like work remaining on CPU and impacting performance).

I also squeezed the memory kind for the CPU into the comments (DDR5), and I am about to add the missing result for the low-scaling GW benchmark (perhaps putting both CUDA and OpenCL based numbers).

**Comment by robertschade** [comment]
Thank you all for contributions and discussions. The slides are now available at https://www.cp2k.org/dev:meetings#notes.
